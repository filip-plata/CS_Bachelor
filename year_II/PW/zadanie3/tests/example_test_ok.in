6 2 4
1 x[0] = (x[1] + x[2])
2 x[2] = ((2 * x[1]) + x[3])
3 x[3] 10 x[1] 3
4 x[0] 5
5 x[2] 1 x[3] 5
6 x[3] 2 x[1] 3
