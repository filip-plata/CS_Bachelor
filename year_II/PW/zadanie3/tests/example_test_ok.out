1 P
2 P
3 P 19
4 P 5
5 F
6 P 11
